### ISSUE TITLE
Accessibility: Home page highlighting elements

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#221]
I have an error with the SiteImprove chrome extension "highlighting elements" on the home page for:

- the 6 dots <button> on the bottom right of the hero image

- the links inside each of the audience buttons: (for instance, the yellow "FUTURE STUDENTS" link inside the future students tab, and all the yellow links inside each tab)

- The two "Read more..." links in the profile section

Suggested fix:
2.4.7 Focus Visible

With keyboard navigation it is important that the focused component can be visually identified. If the focused component is not somehow highlighted, the user cannot be sure which component has focus.

Make sure that user interface elements, that can receive keyboard focus, are highlighted on focus.

**Comment by joelworsham** [comment]
I set all links to have the hover effect when focused as well. This should clear this up. I also set a focus state for the slider dots.

**Comment by banksashleym** [comment]
Hi Joel, sorry to bug you about this one but the error is still showing up about highlighting elements?
![capture](https://user-images.githubusercontent.com/15264315/36268577-d743e4ba-1244-11e8-83ad-d257e5dadc84.JPG)

**Comment by joelworsham** [comment]
I think the issue is that the way I'm "highlighting" it isn't the way the checker expects, but as you can see in the screenshot, it is absolutely highlighted on focus (same is true for read more):

!()[https://d.pr/i/ughhUh]

I also don't think this error is referring to the navigation tabs, but rather just to the links inside the tabs.

So, this is absolutely meeting criteria for accessibility. But if it is absolutely critical for you to have this specific checker validating it, I can try to make some adjustments that the user won't see, but may satisfy the checker.

**Comment by banksashleym** [comment]
I agree, I think a lot of these issues are being fixed more to satisfy the error checker. If you could try to do a workaround to eliminate the error that would be great. I have read that when an auditing company audits a website they start with a scan - so if we can eliminate any errors right off the bat it would be better than having them have to manually check into errors (if they even do that). Thanks!

**Comment by joelworsham** [comment]
Thank you! I've opened 2 more issues which should address the above: #232 #233

**Comment by banksashleym** [comment]
Great!! I really appreciate it!

Have a great weekend ☺

Ashley Banks
Graphic/Web Designer
P  517.796.8465
F  517.796.8633
E  ABanks@jccmi.edu<mailto:ABanks@jccmi.edu>

[90 logo email signature]

Jackson College
2111 Emmons Road
Jackson, MI 49201
